datacube/storage/storage.py

```diff
@@ -55,6 +55,11 @@
                    if str(rasterio.__gdal_version__) >= '1.10.0' else
                    'NETCDF_DIMENSION_')
 
+# rasterio 1.0aX series had warp bug until 1.0a9 So we use workaround code for
+# these versions only, because of silly numbering scheme we can't use simple
+# comparison operators, since '1.0a10' is less than '1.0a9'
```

Comment:
There are two vaguely built in ways to compare version numbers. Unfortunately they each have drawbacks... so I'm happy with this. :)

https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python